solc/CommandLineInterface.cpp

```diff
@@ -790,7 +799,7 @@ void CommandLineInterface::handleCombinedJSON()
 	{
 		Json::Value contractData(Json::objectValue);
 		if (requests.count(g_strAbi))
-			contractData[g_strAbi] = dev::jsonCompactPrint(m_compiler->contractABI(contractName));
```

Comment:
While I like this way better, there might be others using the combined JSON as is and this is a breaking change for them. Probably we should do it separately, or rather encourage others to use the *standard json* interface instead.